docs/admin/provisioners.md

```diff
@@ -47,50 +47,91 @@ the [Helm example](#example-running-an-external-provisioner-with-helm) below.
+coder templates push on-prem \
+  --provisioner-tag environment=on_prem
+
+# Or, match the provisioner exactly
+coder templates push on-prem-chicago \
+  --provisioner-tag environment=on_prem \
+  --provisioner-tag data_center=chicago
+```
+
+A provisioner can run a given build job if one of the below is true:
+
+1. The provisioner and job tags are both organization-scoped and both have no
+   additional tags set,
+1. The set of tags of the build job is a subset of the set of tags of the
+   provisioner.
+
+This is illustrated in the below table:
+
+| Provisioner Tags                                                                      | Job Tags                                                                  | Can run job? |
+| ------------------------------------------------------------------------------------- | ------------------------------------------------------------------------- | ------------ |
+| `{"owner":"","scope":"organization"}`                                                 | `{"owner":"","scope":"organization"}`                                     | true         |
+| `{"owner":"","scope":"organization"}`                                                 | `{"environment":"on_prem","owner":"","scope":"organization"}`             | false        |
+| `{"environment":"on_prem","owner":"","scope":"organization"}`                         | `{"owner":"","scope":"organization"}`                                     | false        |
+| `{"environment":"on_prem","owner":"","scope":"organization"}`                         | `{"foo":"bar","owner":"","scope":"organization"}`                         | true         |
+| `{"environment":"on_prem","owner":"","scope":"organization"}`                         | `{"data_center":"chicago","foo":"bar","owner":"","scope":"organization"}` | false        |
+| `{"data_center":"chicago","environment":"on_prem","owner":"","scope":"organization"}` | `{"foo":"bar","owner":"","scope":"organization"}`                         | true         |
+| `{"data_center":"chicago","environment":"on_prem","owner":"","scope":"organization"}` | `{"data_center":"chicago","foo":"bar","owner":"","scope":"organization"}` | true         |
+| `{"owner":"aaa","scope":"owner"}`                                                     | `{"owner":"","scope":"organization"}`                                     | false        |
+| `{"owner":"aaa","scope":"owner"}`                                                     | `{"owner":"aaa","scope":"owner"}`                                         | true         |
+| `{"owner":"aaa","scope":"owner"}`                                                     | `{"owner":"bbb","scope":"owner"}`                                         | false        |
+| `{"owner":"","scope":"organization"}`                                                 | `{"owner":"aaa","scope":"owner"}`                                         | false        |
+
+The external provisioner in the above example can run build jobs with tags:
```

Comment:
which is the above example? The one on line 86?  If so, this should be moved to before the table